pkg/controller/servicebinding/reconciler_test.go

```diff
@@ -368,7 +368,7 @@ func TestApplicationNotFound(t *testing.T) {
 	requireConditionPresentAndTrue(t, InjectionReady, sbrOutput2.Status.Conditions)
 	requireConditionPresentAndTrue(t, BindingReady, sbrOutput2.Status.Conditions)
 
-	require.Equal(t, reconcilerName, sbrOutput2.Status.Secret)
+	require.Contains(t, sbrOutput2.Status.Secret, reconcilerName)
```

Comment:
Please include the checksum in the validation; it should be thought as a contract that the secret is named as expected.

`Contains` might match different resources, since it has no boundaries.

Reply:
Okay

Reply:
Done here cfb3675